Assets/Scripts/Models/Character.cs

```diff
@@ -874,8 +876,60 @@ public IEnumerable<ContextMenuAction> GetContextMenuActions(ContextMenu contextM
         yield return new ContextMenuAction
         {
             Text = "Poke "+GetName(),
-            RequiereCharacterSelected = false,
+            RequireCharacterSelected = false,
             Action = (cm, c) => Debug.ULogChannel("Character", GetDescription())
         };
     }
+
+    public void PrioritizeJob(Job j)
+    {
+        AbandonJob(false);
+        World.current.jobQueue.Remove(j);
+        j.IsBeingWorked = true;
+        //check if the character is carrying any materials and if they could be used for the new job,
+        //if the character is carrying materials but is not used in the new job, then drop them
+        //on the current tile for now.
```

Comment:
Between comment slashes '//' and comment should be a white space.
Also, comments should start with a capital letter.